README.md

```diff
@@ -2,7 +2,7 @@
 
 # Syncstorage-rs
 
-Mozilla Sync Storage built with [Rust](https://rust-lang.org).
+Mozilla Sync Storage built with [Rust](https://rust-lang.org) on [GCP](https://cloud.google.com/).
```

Comment:
Sync storage can run on GCP, but ideally, does not require it. We had to focus on GCP first. There's ongoing work to get sync working on systems outside of GCP.

Reply:
Agreed, so closing this one out